system/hardware/tici/hardware.py

```diff
@@ -211,8 +211,8 @@ def get_imei(self, slot):
     return str(self.get_modem().Get(MM_MODEM, 'EquipmentIdentifier', dbus_interface=DBUS_PROPS, timeout=TIMEOUT))
 
   def get_network_info(self):
-    modem = self.get_modem()
     try:
+      modem = self.get_modem()
```

Comment:
if this isn't try catched, it causes the whole hardware state to not send when the modem isn't available (after a restart for example)